app/assets/javascripts/discourse/app/helpers/unique-id.js

```diff
@@ -0,0 +1,10 @@
+// https://github.com/emberjs/ember.js/blob/master/packages/@ember/-internals/glimmer/lib/helpers/unique-id.ts
+export default function uniqueId() {
```

Comment:
Not sure if this is necessary, can we just use  `import guid from "pretty-text/guid";` ?

Reply:
We also seem to have `import { guidFor } from "@ember/object/internals";` ... maybe that could be used too? So many unique ways of making unique IDs :sweat_smile:

Reply:
`guidFor` needs an object like a DOM node which I don't have here, I think it's just easier to have this here